tests/test_xgboost_converter.py

```diff
@@ -263,6 +265,47 @@ def test_xgb_classifier_converter_torchscript(self):
             self.assertIsNotNone(torch_model)
             np.testing.assert_allclose(model.predict_proba(X), torch_model.predict_proba(X), rtol=1e-06, atol=1e-06)
 
+    # TVM backend tests.
+    # TVM backend regression.
+    @unittest.skipIf(not xgboost_installed(), reason="XGBoost test requires XGBoost installed")
+    @unittest.skipIf(not tvm_installed(), reason="TVM test requires TVM installed")
+    def test_xgb_regressor_converter_tvm(self):
+        warnings.filterwarnings("ignore")
+        import torch
+
+        for max_depth in [1, 3, 8, 10, 12]:
+            model = xgb.XGBRegressor(n_estimators=10, max_depth=max_depth)
+            np.random.seed(0)
+            X = np.random.rand(100, 200)
+            X = np.array(X, dtype=np.float32)
+            y = np.random.randint(1000, size=100)
+
+            model.fit(X, y)
+
+            torch_model = hummingbird.ml.convert(model, "torchscript", X, extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+            self.assertIsNotNone(torch_model)
+            np.testing.assert_allclose(model.predict(X), torch_model.predict(X), rtol=1e-06, atol=1e-06)
+
+    # Test TVM backend classification.
+    @unittest.skipIf(not xgboost_installed(), reason="XGBoost test requires XGBoost installed")
+    @unittest.skipIf(not tvm_installed(), reason="TVM test requires TVM installed")
+    def test_xgb_classifier_converter_tvm(self):
+        warnings.filterwarnings("ignore")
+        import torch
+
+        for max_depth in [1, 3, 8, 10, 12]:
+            model = xgb.XGBClassifier(n_estimators=10, max_depth=max_depth)
+            np.random.seed(0)
+            X = np.random.rand(100, 200)
+            X = np.array(X, dtype=np.float32)
+            y = np.random.randint(2, size=100)
+
+            model.fit(X, y)
+
+            torch_model = hummingbird.ml.convert(model, "tvm", X, extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
```

Comment:
maybe `tvm_model` not `torch_model`?  same as line 285

Reply:
Yep.